### DISCUSSION TITLE
Installation roadmap

### DISCUSSION THREAD

**Discussion by danielo37** [Security-Onion-Solutions/securityonion#12123] [Unsupported Versions] [answered]
Hello there,

I would like to know if anyone can share something like a roadmap, checklist, installation steps to follow to know that we installed the SO correctly and we get the most out of it, and make sure all we have on SO is integrated to the production environment.

Something like [ProductionDeployment](https://github.com/Security-Onion-Solutions/security-onion/wiki/ProductionDeployment).

Thank you in advance.

**Comment by reyesj2** [comment, marked as answer]
The docs best practices section has a good overview. 
https://docs.securityonion.net/en/2.4/best-practices.html?highlight=best%20practice#best-practices

Our YouTube channel has some good videos about Security Onion deployment and configuration. https://www.youtube.com/@security-onion/videos
